packages/cli/src/commands/create/index.ts

```diff
@@ -1,45 +1,15 @@
 import { createMonorepo } from './monorepo'
 import { createStandalone } from './standalone'
-import { Structure } from '../add'
 import {
-  betterPrompts,
   isValidPackageManager,
   selectPackageManager,
+  selectRepositoryStructure,
+  isValidStructure,
 } from '../../utils'
 import { CreateFunc, CreateArgs } from './types'
-import { PackageManager } from '../types'
 
 export type { CreateArgs }
 
-const selectRepositoryStructure = async () => {
```

Comment:
This has been moved to `utils` with no change. I've moved it to easily mock it later.